recipes-graphics/xorg-driver/xf86-video-intel_git.bb

```diff
@@ -1,46 +0,0 @@
-require recipes-graphics/xorg-driver/xorg-driver-video.inc
-
-SUMMARY = "X.Org X server -- Intel integrated graphics chipsets driver"
-
-DESCRIPTION = "intel is an Xorg driver for Intel integrated graphics \
-chipsets. The driver supports depths 8, 15, 16 and 24. On some chipsets, \
-the driver supports hardware accelerated 3D via the Direct Rendering \
-Infrastructure (DRI)."
-
-LIC_FILES_CHKSUM = "file://COPYING;md5=8730ad58d11c7bbad9a7066d69f7808e"
-
-PV = "2.99.917+git${SRCPV}"
-
-SRCREV = "a1a0f76e55d8dfbbf27a288b71b4bd3edbfbe836"
```

Comment:
@koenkooi this recipe is in meta-oe but I'm unsure about patch you made, is it already up-streamed? This have 3 years without modification so I think it's pretty outdated by now?